core/slice.go

```diff
@@ -279,13 +274,14 @@ func (sl *Slice) backfillPETXs(header *types.Header, subManifest types.BlockMani
 }
 
 // relayPh sends pendingHeaderWithTermini to subordinates
-func (sl *Slice) relayPh(pendingHeaderWithTermini types.PendingHeader, updateMiner bool, reorg bool, domOrigin bool, location common.Location) {
+func (sl *Slice) relayPh(pendingHeaderWithTermini types.PendingHeader, reorg bool, domOrigin bool, location common.Location) {
 	nodeCtx := common.NodeLocation.Context()
 
 	if nodeCtx == common.ZONE_CTX {
-		if updateMiner {
-			sl.phCache[sl.pendingHeaderHeadHash].Header.SetLocation(common.NodeLocation)
```

Comment:
(nit) iiuc `pendingHeaderHeadHash` is the hash of the termini that keys the "best" pending header, right? Can we rename it accordingly? Maybe `bestPhKey`? or `bestPhId`? As I understand it

Reply:
Renamed to bestPhKey